scripts/mutect2_wdl/unsupported/mutect2_opt.wdl

```diff
@@ -223,44 +293,44 @@ task M2 {
   File? gnomad
   File? gnomad_index
   String output_vcf_name
-  String m2_docker
   File? gatk4_jar_override
-  Int preemptible_attempts
   String? m2_extra_args
   Boolean? is_bamOut
-  
+
+  # Runtime parameters
+  String m2_docker
+  Int? preemptible_attempts
+  Int disk_space_gb
+  Int? mem
+
+  # Mem is in units of GB but our command and memory runtime values are in MB
+  Int machine_mem = if defined(mem) then mem * 1000 else 3500
+  Int command_mem = machine_mem - 500
+
   # Do not populate this unless you know what you are doing...
   File? auth
 
   command <<<
-
   if [[ "${auth}" == *.json ]]; then
     gsutil cp ${auth} /root/.config/gcloud/application_default_credentials.json
     GOOGLE_APPLICATION_CREDENTIALS=/root/.config/gcloud/application_default_credentials.json
     export GOOGLE_APPLICATION_CREDENTIALS=/root/.config/gcloud/application_default_credentials.json
   fi
 
   # Use GATK Jar override if specified
-  GATK_JAR=${gatk4_jar}
-  if [[ "${gatk4_jar_override}" == *.jar ]]; then
-      GATK_JAR=${gatk4_jar_override}
-  fi
+  GATK_JAR=${default="/root/gatk.jar" gatk4_jar_override}
 
-  if [[ "_${normal_bam}" == *.bam ]]; then
-      java -Xmx4g -jar $GATK_JAR GetSampleName -I ${normal_bam} -O normal_name.txt
-      normal_command_line="-I ${normal_bam} -normal `cat normal_name.txt`"
-  else
-      # Note that normal_name.txt is always created, it's just empty if no normal sample was given.
-      #     This is done to allow an output parameter of the normal sample.
-      touch normal_name.txt
-  fi
+  # These two following commands will only be executed if normal_bam is defined, otherwise they will be blank
```

Comment:
Does `they` here mean `normal_name.txt`?

Reply:
they means the command themselves, like they will be blank in the exec.sh.  we can talk faces if thats still confusing

Reply:
That makes sense